### DISCUSSION TITLE
How can I run facial comparison

### DISCUSSION THREAD

**Discussion by victorn3ss** [community/community#49545] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm working on a project about face comparison, please provide the documentation.

https://github.com/Amir-Shamsi/facial-comparison-api-server

**Comment by otter-computer** [comment]
Welcome to the GitHub Community, @victorn3ss, we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
